src/renderer/components/commands.tsx

```diff
@@ -36,15 +36,15 @@ export const Commands = observer(
 
     public render() {
       const { appState } = this.props;
-      const { isBisectCommandShowing, title } = appState;
+      const { isBisectCommandShowing, title, isSettingsShowing } = appState;
 
       return (
         <div
-          className={
+          className={`${
             window.ElectronFiddle.platform === 'darwin'
               ? 'commands is-mac'
               : 'commands'
-          }
+          }${isSettingsShowing ? ' tabbing-hidden' : ''}`}
```

Comment:
This can be simplified using the `classnames` package which we use in some other components:

```tsx
className={classNames(
  'commands',
  { 'is-mac': window.ElectronFiddle.platform === 'darwin' },
  { 'tabbing-hidden': isSettingsShowing },
)}
```

Reply:
Oh, neat! Thanks for the pointer